### PR TITLE
docs: REST API docs fixes

### PR DESCRIPTION
**Pull request by neilconway**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Comment by justin-determined-ai** [comment]
@hamidzr can you add your fix in #1307 to here?

**Comment by hamidzr** [comment]
> @hamidzr can you add your fix in #1307 to here?

I don't think I can push to @neilconway 's fork

File: docs/how-to/rest-apis.txt [hunk 1]
```diff
@@ -24,11 +26,12 @@ Determined WebUI or by visiting the path `/swagger-ui` on your Determined cluste
...
 
 
-The Swagger UI gives you an up-to-date view of what APIs your Determined cluster is
-providing in a readable and interactive fashion.
+The Swagger UI gives you an up-to-date view of the APIs your Determined cluster
+provides in a readable and interactive fashion.
+
 On this page, you'll see a list of all the available endpoints grouped by their workflows.
 If you click on each endpoint you'll be presented with expected inputs and outputs for it
-as well as an interactive tool that let's you try out the endpoint. Note that for the
```
Comment (hamidzr) [inline docs/how-to/rest-apis.txt:31]: always gets me :facepalm:

Comment (justin-determined-ai) [reply docs/how-to/rest-apis.txt:31]: it's a doozy :)

**Comment by neilconway** [comment]
@hamidzr @justin-determined-ai I added the fix to remove the reference to the private repo. For future reference you should be able to push to my fork?

**Review by justin-determined-ai — APPROVED** [review]
LGTM!

**Review by vishnu2kmohan — CHANGES_REQUESTED** [review]
Minor suggestions.

**Review by vishnu2kmohan — APPROVED** [review]

File: docs/how-to/rest-apis.txt [hunk 2]
```diff
@@ -54,9 +57,9 @@ Sample request:
 
 .. code:: bash
 
-  curl -s "${host}/api/v1/auth/login" \
+  curl -s "${DET_MASTER}/api/v1/auth/login" \
     -H 'Content-Type: application/json' \
-    --data-binary '{"username":"john","password":"2k34h5@$"}' \
+    --data-binary '{"username":"determined","password":""}'
```
Comment (hamidzr) [inline docs/how-to/rest-apis.txt:62]: should we mention that the password is salted and (sha512) hashed? would we provide the password salt here?

Comment (hamidzr) [reply docs/how-to/rest-apis.txt:62]: going w/ an empty password for now :heavy_check_mark:

**Comment by hamidzr** [comment]
generated docs: https://148741-253846879-gh.circle-artifacts.com/0/docs/site/html/index.html

**Review by hamidzr — APPROVED** [review]

**Pull request merged**